thoth/glyph/lib.py

```diff
@@ -0,0 +1,110 @@
+import logging
```

Comment:
Please add a license file with copyright to each source code file.

Reply:
Done